Services/SciMaterials.Services.API/Services/Files/Stores/FileSystemStore.cs

```diff
@@ -102,10 +102,6 @@ public void Delete(string filePath)
     {
         if (File.Exists(filePath))
             File.Delete(filePath);

```

Comment:
```suggestion
        File.Delete(filePath);
        
```
Нет необходимости предварительно проверять наличие файла.